resources/assets/uad_lists.json

```diff
@@ -17903,7 +17903,7 @@
   {
     "id": "com.android.vending",
     "list": "Google",
-    "description": "Google Play Store",
+    "description": "Google Play Store\nWARNING: Uninstalling Google Play Store and using alternative sources like Aurora Store or F-droid for app installation or updates may cause issues.\nApplications may get stuck at the installing or updating screen.\nTo be able to update or install through alternatives, you might need to keep this app installed.",
```

Comment:
Shouldn't you mention this only started happening since Android 13?

Also, maybe change the `removal` from `expert` to `advanced`? What do you think?

Reply:
Sure, I will change the description to mention the Android version. As for the removal, I still think it should be left at expert because not being able to update or install apps on your phone can have a significant impact.

Reply:
So you mean that you can't update any app via alternative app stores when Google Play Store is not installed? In that case: yeah, fair. Also, does it only happen with 13? What about 10, 11 and 12?

Reply:
yeah, exactly. you cannot update/install without google play store and it only happened to me on android 13. I don't know about other versions since I don't have access to any phones with older versions of android right now.

Reply:
What ROM are you using? Seems to work fine for me (F-droid, Neo Store, Aurora Store). Maybe it varies OEM to OEM?
Tested on OnePlus 9RT - Android 11 & 13 (OxygenOS)